nuttx-configs/aerofc-v1/nsh/defconfig

```diff
@@ -536,9 +536,9 @@ CONFIG_STM32_I2CTIMEOTICKS=10
 #
 # Configuration
 #
-CONFIG_STM32_BBSRAM=y
-CONFIG_STM32_BBSRAM_FILES=4
-CONFIG_STM32_SAVE_CRASHDUMP=y
+# CONFIG_STM32_BBSRAM is not set
+# CONFIG_STM32_BBSRAM_FILES is not set
+# CONFIG_STM32_SAVE_CRASHDUMP is not set
 # CONFIG_STM32_HAVE_RTC_COUNTER is not set
 # CONFIG_STM32_HAVE_RTC_SUBSECONDS is not set
```

Comment:
On Some targets the saving of the fault data to the BBSRAM should be independent of having an SD card to log it to. So I am not sure what cause the loop. Did you identify the root cause?